api/src/pcapi/routes/native/v1/serialization/bookings.py

```diff
@@ -32,6 +31,20 @@ class BookOfferResponse(BaseModel):
     bookingId: int
 
 
+class BookingVenueResponseGetterDict(GetterDict):
+    def get(self, key: str, default: Any | None = None) -> Any:
+        if key == "coordinates":
+            return Coordinates(latitude=self._obj.latitude, longitude=self._obj.longitude)
+
+        if key == "address":
+            return self._obj.street
+
+        if key == "name":
+            return self._obj.publicName or self._obj.name
```

Comment:
Tu peux utiliser la propriété `common_name` existante, qui fait exactement ce que tu veux.
Ici et dans d'autres fichiers plus bas dans cette PR.
```suggestion
            return self._obj.common_name
```